clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -459,9 +459,6 @@ def warn_drv_assuming_mfloat_abi_is : Warning<
 def warn_drv_unsupported_float_abi_by_lib : Warning<
   "float ABI '%0' is not supported by current library">,
   InGroup<UnsupportedABI>;
-def warn_drv_no_floating_point_registers: Warning<
```

Comment:
This is a build break for anyone that is explicitly referring to this warning.

I'm not sure what the protocol for removing warnings is in clang. For example are they release noted/deprecated or just left in and ignored?

Reply:
`-Wunsupported-abi` still exists (it is a group containing several warnings). The only thing that is removed here is the particular manifestation of this warning for the ARM target. It should only be a build breakage if `-Wunsupported-abi` were removed completely.